tests/test_approval_token_security.py

```
@@ -0,0 +1,223 @@
+            "tool_calls": [
+                {
+                    "id": "tc_happy",
+                    "type": "function",
+                    "function": {"name": "bash", "arguments": args},
+                    "pending_approval": True,
+                    "approval_ticket": ticket,
+                }
+            ],
+        },
+    ]
+    msgs, events = ai._execute_tool_decisions(
```

Ruff (0.15.17)

[warning] Unpacked variable `events` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)